src/states/features/users/usersSlice.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-use-before-define */
```

Comment:
Please remove these eslint-disable comments

Reply:
For the padding and pagination
https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/72489391/0921edb1-5ff1-4eb1-addd-6075a8f1fc20